tensorflow/contrib/rnn/python/ops/rnn_cell_test.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
```

Comment:
we have a test in contrib/rnn/python/kernel_tests/rnn_cell_test.  you should add your test class there instead of here.